scripts/nn/test/compare_backends/gen_conv2d.dml

```diff
@@ -0,0 +1,25 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+X = rand(rows=$N, cols=$C*$H*$W, sparsity=$sp, min=-0.5, max=1)
+w = rand(rows=$K, cols=$C*$R*$S, sparsity=$sp, min=-0.5, max=1)
```

Comment:
Can we update the filter size variables here and elsewhere to match those used elsewhere in the library?  Specifically, `K` -> `F`, `R` -> `Hf`, and `S` -> `Wf`.  I think this would really help for others in the future who come to run these tests and reason about them.